browser/src/Config.ts

```diff
@@ -268,6 +268,30 @@ export class Config extends EventEmitter {
         this._activateIfOniObjectIsAvailable()
     }
 
+    private _applyDefaultKeyBindings(oni: Oni.Plugin.Api): void {
+        oni.input.unbindAll()
+
+        if (Platform.isLinux() || Platform.isWindows()) {
+            if (this.getValue("editor.clipboard.enabled")) {
+                oni.input.bind("<C-c>", "editor.clipboard.yank", () => oni.editors.activeEditor.mode === "visual")
+                oni.input.bind("<C-v>", "editor.clipboard.paste", () => oni.editors.activeEditor.mode === "insert")
+            }
+        } else {
+            if (this.getValue("editor.clipboard.enabled")) {
+                oni.input.bind("<M-c>", "editor.clipboard.yank", () => oni.editors.activeEditor.mode === "visual")
+                oni.input.bind("<M-v>", "editor.clipboard.paste", () => oni.editors.activeEditor.mode === "insert")
+            }
+        }
```

Comment:
You can simplify this to
```ts
const prefix = Platform.isLinux() || Platform.isWindows() ? "C" : "M"
if (this.getValue("editor.clipboard.enabled")) {
    oni.input.bind("<"+prefix+"-c>", "editor.clipboard.yank", () => oni.editors.activeEditor.mode === "visual")
    oni.input.bind("<"+prefix+"-v>", "editor.clipboard.paste", () => oni.editors.activeEditor.mode === "insert")
}
```
this can also server as example for users who use both mac and windows, if they want on config file they can use this syntax so they don't have to have different config files.